umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/provisioning.md

```diff
@@ -1,8 +1,49 @@
 ---
 title: Provisioning
 menuTitle: Provisioning
-description:
+description: |
+  Discover how to provision both the Data and the Device & Container
+  Infrastructures using the Management Console.
 weight: 1000
 edition: community
-draft: true
+draft: false
 ---
+
+The Management Console simplifies the deployment of the
+[Data Infrastructure](/docs/architecture/data-infrastructure/) on any existing
+system, and also guides you through the provisioning of the
+[Device & Container Infrastructure](/docs/architecture/device--container-infrastructure/).
+
+## When should I use it?
+
+Utilize the Management Console for initial deployment of the United
+Manufacturing Hub. It supports provisioning on both centralized servers and
```

Comment:
Whenever you want to leverage the Data Infrastructure of the UMH. The Provisioning feature will help you in setting up in installing an operating system (if you don't have any yet), will help you in getting a Kubernetes setup on that OS, and will install the Data Infrastructure including the "agent" / companion. See also architecture for more information.